src/front-end/src/components/AdminSignup/AdminSignup.js

```diff
@@ -51,7 +55,9 @@ class AdminSignup extends Component {
             onFinish={this.onFinish}
             onFinishFailed={this.onFinishFailed}
             autoComplete="off"
+            // Used to adjust the placement of labels
```

Comment:
I generally prefer comments that make no reference to the code itself and use imperative tone, I.e.: 
```suggestion
            // Adjust the placement of labels
```